src/api/main.rs

```diff
@@ -1,18 +1,37 @@
+#[macro_use] 
+extern crate diesel;
+extern crate dotenv;
+extern crate chrono;
```

Comment:
Not needed here.

Reply:
good catch, thanks! removed dotenv and chrono, but diesel is still needed though as it has to be declared crate-wide in the root file